example/src/main/java/io/honeycomb/opentelemetry/android/example/MainActivity.kt

```diff
@@ -172,6 +180,11 @@ fun Playground(otel: OpenTelemetryRum?, modifier: Modifier = Modifier) {
                 text = "Send Span",
             )
         }
+        Button(onClick = { onSendMetrics(otel) }) {
```

Comment:
Should we add a smoke test for this? I know we aren't worried about the logging part, but I wonder if it would be useful to verify metrics still get captured. I could be convinced either way.

Reply:
Yeah I'll add one